app-react/src/main/kotlin/com/ustadmobile/view/ScopedGrantEditScreen.kt

```diff
@@ -0,0 +1,66 @@
+package com.ustadmobile.view
+
+import com.ustadmobile.core.generated.locale.MessageID
+import com.ustadmobile.core.hooks.useStringsXml
+import com.ustadmobile.core.model.BitmaskFlag
+import com.ustadmobile.core.viewmodel.ScopedGrantEditUiState
+import mui.material.*
+import react.*
+
+external interface ScopedGrantEditScreenProps : Props {
+
+    var uiState: ScopedGrantEditUiState
+
+    var onChangedBitmask: (BitmaskFlag?) -> Unit
+
+}
+
+val ScopedGrantEditScreenComponent2 = FC<ScopedGrantEditScreenProps> { props ->
+
+    val strings = useStringsXml()
+
+    Container {
+
+        List{
+            props.uiState.bitmaskList
+                .forEach { bitmask ->
+                ListItem{
```

Comment:
Please use ListItemButton and it's onClick function so that clicking anywhere on the item will toggle the switch.